tools/src/format.rs

```diff
@@ -872,7 +890,7 @@ fn generate_commit_info(tree_name: &str,
             F::T(format!("<tr><td>commit</td><td>{}</td></tr>", format_rev(tree_name, commit.id()))),
             F::Seq(parents),
             F::Seq(hg),
-            F::T(format!("<tr><td>git</td><td>{}</td></tr>", gitstr)),
+            F::T(gitstr.map_or(String::new(), |g| format!("<tr><td>git</td><td>{}</td></tr>", g))),
```

Comment:
I guess ideally there shouldn't be an empty entry, but shrug.

Reply:
I left it this way for aesthetic reasons... didn't want to insert an if condition to drop the `F::T` entirely.